File: specification/specs/indices/index_settings/index_templates/get_index_template/GetIndexTemplateRequest.ts
@rest_spec_name("indices.get_template")
class GetIndexTemplateRequest extends RequestBase {
  path_parts?: {
    name?: Names;
  }
  query_parameters?: {
    flat_settings?: boolean;
    include_type_name?: boolean;
    local?: boolean;
    master_timeout?: Time;
  }
  body?: {
  }
}

File: output/csharp/Indices/IndexSettings/IndexTemplates/GetIndexTemplate/GetIndexTemplateRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Indices {

	public class GetIndexTemplateRequest : RequestBase {
		
		[DataMember(Name="flat_settings")]
		public bool FlatSettings { get; set; }


		[DataMember(Name="include_type_name")]
		public bool IncludeTypeName { get; set; }


		[DataMember(Name="local")]
		public bool Local { get; set; }


		[DataMember(Name="master_timeout")]
		public Time MasterTimeout { get; set; }

	}
}
